deploy/charts/rook-ceph-cluster/templates/deployment.yaml

```diff
@@ -86,7 +86,6 @@ spec:
               # create the keyring file
               cat <<EOF > ${KEYRING_FILE}
               [${ROOK_CEPH_USERNAME}]
-              key = ${ROOK_CEPH_SECRET}
```

Comment:
Good catch, I see the [toolbox.yaml](https://github.com/rook/rook/blob/release-1.13/deploy/examples/toolbox.yaml#L79-L81) example does not have this issue.